CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Views/Main/HomeView.swift

```diff
@@ -8,30 +8,28 @@
 import SwiftUI
 
 struct HomeView: View {
-    @State var hasScrolled = false
     @Namespace var namespace
     @State var show = false
     @State var showStatusBar = true
     @State var selectedID = UUID()
     @State var showCourse = false
     @State var selectedIndex = 0
     @EnvironmentObject var model: Model
-    
+    @AppStorage("isLiteMode") var isLiteMode = true
```

Comment:
What is `LiteMode`?

Reply:
i think it makes app simpler ( removes animations and other stuff)

Reply:
@Forkipe @sghourab can you share documentation with me?

Reply:
@Den-Ree It's not a library, it's a Bool stored in AppStorage that we used throughout the app with the custom Modifier 'ifMode' to alter some views: 

extension View {
@ViewBuilder func ifMode<Content: View>(_ condition: Bool, transform: (Self) -> Content) -> some View {
        if condition {
            transform(self)
        } else {
            self
        }
    }}

And this can then be applied on any view, example: 
Rectangle
              .ifMode(!isLiteMode, transform: { view in
            view
                .shadow(color: AppColors.shadow.opacity(0.5), radius: 30, x: 0, y: 15)
        })

OR simply like this:

 Section {
                    Toggle(isOn: $isLiteMode) {
                        Label("Lite Mode", systemImage: isLiteMode ? "tortoise" : "hare")
                    }
                }